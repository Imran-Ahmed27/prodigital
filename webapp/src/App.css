.banner {
  background-image: url("./assets/banner.jpg");
}
