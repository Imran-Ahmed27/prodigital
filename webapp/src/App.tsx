import "./App.css";
import { Outlet, ReactLocation, Router } from "@tanstack/react-location";
import { LocationGenerics, routes } from "./routes";
import MainLayout from "./layouts/MainLayout";
import { ToastContainer } from "react-toastify";
import "react-toastify/dist/ReactToastify.min.css";

const location = new ReactLocation<LocationGenerics>();

function App() {
  return (
    <>
      <Router location={location} routes={routes}>
        <MainLayout>
          <Outlet />
        </MainLayout>
      </Router>
      <ToastContainer position="top-center" />
    </>
  );
}

export default App;
