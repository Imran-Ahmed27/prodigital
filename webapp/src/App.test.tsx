import { describe, expect, it } from "vitest";
import { render } from "@testing-library/react";
import App from "./App";

describe("App", () => {
  it("Renders correctly", () => {
    const screen = render(<App />);
    expect(screen.container).toMatchSnapshot();
  });
});
