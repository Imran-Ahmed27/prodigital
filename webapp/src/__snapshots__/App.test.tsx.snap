// Vitest Snapshot v1

exports[`App > Renders correctly 1`] = `<div />`;
